python/lsst/daf/butler/registry/queries/expressions/parser/exprTree.py

```diff
@@ -488,6 +492,115 @@ def __str__(self) -> str:
         return f"POINT({self.ra}, {self.dec})"
 
 
+class CircleNode(Node):
+    """Node representing a circle, (ra, dec, radius) pair.
+
+    Parameters
+    ----------
+    ra : `Node`
+        Node representing circle center ra value.
+    dec : `Node`
+        Node representing circle center dec value.
+    radius : `Node`
+        Node representing circle radius value.
+    """
+
+    def __init__(self, ra: Node, dec: Node, radius: Node):
+        super().__init__((ra, dec, radius))
+        self.ra = ra
+        self.dec = dec
+        self.radius = radius
+
+    def visit(self, visitor: TreeVisitor) -> Any:
+        # Docstring inherited from Node.visit
+        ra = self.ra.visit(visitor)
+        dec = self.dec.visit(visitor)
+        radius = self.radius.visit(visitor)
+        return visitor.visitCircleNode(ra, dec, radius, self)
+
+    def __str__(self) -> str:
+        return f"CIRCLE({self.ra}, {self.dec}, {self.radius})"
```

Comment:
I see none of the stringifications are tested -- is it easy to force them to be stringified somewhere? I assume you added the string form for a reason.